src/Sarif.Multitool/CommandUtilities.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    internal static class CommandUtilities
+    {
+        internal static string GetTransformedOutputFileName(SingleFileOptionsBase options)
```

Comment:
Note to self: See if there are unit tests for this method.

Reply:
Ok, I see there are none. Please add.

Reply:
Oh... this is an existing method that moved from one file to another. Never mind.

Reply:
I'll bug this and use it to fill downtime...

Reply:
#909